app/ember/src/client/preview/render.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable no-undef */
+import { window, document } from 'global';
+import { stripIndents } from 'common-tags';
+import { getApplicationConfig } from './utils';
+
+const config = getApplicationConfig();
+
+const rootEl = document.getElementById('root');
+
+const app = window.require(`${config.APP.name}/app`).default.create({
+  autoboot: false,
+  rootElement: rootEl,
+});
+
+let appInstance;
+
+function render(options, el) {
```

Comment:
This is _essentially_ the same as how component tests are setup and ran. I'd like to extract a primitive library that can be shared between the main `@ember/test-helpers` and this project so that we can more easily support broad ranges of Ember applications (and reduce specific coupling in storybook itself).

Thoughts?

Reply:
I totally think this would be something we could decouple. Something like `@ember/setup` that would just handle setting up and exposing a booted application